rskj-core/src/main/java/co/rsk/util/HexUtils.java

```diff
@@ -354,7 +367,7 @@ public static byte[] decode(byte[] dataBytes) {
      */
     public static int jsonHexToInt(final String param) {
         if (!hasHexPrefix(param)) {
-            throw invalidParamError(INCORRECT_HEX_SYNTAX);
+            throw invalidParamError("invalid argument 1: json: cannot unmarshal hex string without 0x prefix into value of type int");
```

Comment:
Not sure about the error message. In this case you are supposing that the invalid argument will be always in the position 1 (I don't have the certainty of it). 

And as commented above I think we shouldn't return directly an RPC related error here.